src/lib/sharedUtils.js

```diff
@@ -74,3 +74,12 @@ export function format(date, format = 'Pp', config = {}) {
 export function formatDistanceToNow(date, config = {}) {
   return dateFnsFormatDistanceToNow(date, { ...config, locale });
 }
+
+export function addTime(theDate, { seconds, minutes, hours, days }) {
```

Comment:
Since we already use `date-fns` in our application, I think we should use [`date-fns/add`](https://date-fns.org/v2.14.0/docs/add) directly, instead of implementing our own function.